##
# If you put here relative topic names, they will be append to namespace.
# If you put absolute topics, no leading namespace will be added.
##
namespace: 'auk'

# RTK Info topics
piksi_receiver_state_topic: 'piksi/debug/receiver_state'
piksi_uart_state_topic: 'piksi/debug/uart_state'
piksi_baseline_ned_topic: 'piksi/baseline_ned'

# MSF topics and services
msf_odometry_topic: 'msf_core/odometry'
init_msf_scale_srv: 'pose_sensor_rovio/pose_sensor/initialize_msf_scale'
init_msf_height_srv: 'pose_sensor_rovio/pose_sensor/initialize_msf_height'

# Init Rovio service
init_rovio_srv: 'init_rovio_enu/send_reset_to_rovio'
